frontend/src/components/PhaseSelectionBox/PhaseSelectionBox.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from "react";
+import { Phase } from "../../models/phase";
+import { PHASES } from "../../Phases";
+import {
+  getCurrentPhase,
+  setCurrentPhase,
+  setCompletedPhase,
+} from "../../service/phaseService";
+
+import "./PhaseSelectionBox.css";
+
+function PhaseSelectionBox(this: any) {
+  // start on sandbox mode
+  const [currPhase, setCurrPhase] = useState<Phase>(PHASES[3]);
+
+  // valid phases that the user can go to from the current phase
+  const [currValidPhases, setCurrValidPhases] = useState<number[]>([0]);
```

Comment:
Change this to be just a number representing the number of completed phases. This is the same as what's stored in the session on the backend, so the frontend can get this from the backend. `0` would mean no phases are completed, so only phase 0 and sandbox are selectable.